src/Aguacongas.TheIdServer/README.md

```diff
@@ -431,6 +434,31 @@ It this case the client registration request must contains the *contacts* array.
 }
 ```
 
+## Configure Jwt request validator
+
+Tokens returned by request_uri parameter are validated using the rules defined in *TokenValidationParameters* section. By default all this following rule are true.
```

Comment:
The last sentence is confusing. You are saying that the following rules are "true"', but the list of rules shown are all "false".  From what I am understanding, you are saying that rules shown are included by default, unless action is taken to override the rules.  If so, you may want to change the last sentence to the following for clarity:

By default, the following rules are defined.